core/settings.py

```diff
@@ -190,7 +190,7 @@
 
 # Logs
 
-LOG_DIR = BASE_DIR / "logs"
+LOG_DIR = BASE_DIR / os.getenv("LOG_PATH", "logs")
 LOG_DIR.mkdir(exist_ok=True)
```

Comment:
@rex9840  Please conform to the changes requested by @PublisherName as well. Also look into why CI is breaking as well.

Reply:
@n1rjal review as the linting error has been solved